mitxpro/settings.py

```diff
@@ -30,7 +30,7 @@
 
 ALLOWED_HOSTS = ["*"]
 
-SECURE_SSL_REDIRECT = get_bool("MITXPRO_SECURE_SSL_REDIRECT", True)
+SECURE_SSL_REDIRECT = get_bool("MITXPRO_SECURE_SSL_REDIRECT", False)
```

Comment:
Do we really need to disable secure SSL redirects here? Or is this a side-effect of rebasing/merging from master or something?

Reply:
haha. push some un used stuff. let me remove it. Thanks